catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityFilterPlugin.java

```diff
@@ -97,4 +61,53 @@ public void setAttributeMap(List<String> attributeMappings) {
     public void setAttributeMap(Map<String, List<String>> attributeMap) {
         MetacardValidityFilterPlugin.attributeMap = attributeMap;
     }
+
+    @Override
+    public PolicyResponse processPreCreate(Metacard input, Map<String, Serializable> properties)
+            throws StopProcessingException {
+        return new PolicyResponseImpl(new HashMap<>(), new HashMap<>());
+    }
+
+    @Override
+    public PolicyResponse processPreUpdate(Metacard input, Map<String, Serializable> properties)
+            throws StopProcessingException {
+        return new PolicyResponseImpl(new HashMap<>(), new HashMap<>());
+    }
+
+    @Override
+    public PolicyResponse processPreDelete(String attributeName, List<Serializable> attributeValues,
+            Map<String, Serializable> properties) throws StopProcessingException {
+        return new PolicyResponseImpl(new HashMap<>(), new HashMap<>());
+    }
+
+    @Override
+    public PolicyResponse processPreQuery(Query query, Map<String, Serializable> properties)
+            throws StopProcessingException {
+        return new PolicyResponseImpl(new HashMap<>(), new HashMap<>());
+    }
+
+    @Override
+    public PolicyResponse processPostQuery(Result input, Map<String, Serializable> properties)
+            throws StopProcessingException {
+
+        if (input == null || input.getMetacard() == null) {
+            return new PolicyResponseImpl(new HashMap<>(), new HashMap<>());
+        }
+        Metacard metacard = input.getMetacard();
+        HashMap<String, Set<String>> securityMap = new HashMap<>();
+
+        if ((metacard.getAttribute(VALIDATION_ERRORS) != null && metacard.getAttribute(
```

Comment:
It could be but that would make for one huge conditional. I think it's more readable as is but if you feel strongly about combining them I can look into it more.